cpp/oneapi/dal/algo/svm/backend/cpu/train_kernel_thunder.cpp

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <daal/src/algorithms/svm/svm_train_thunder_kernel.h>
+
+#include "oneapi/dal/algo/svm/backend/cpu/train_kernel.hpp"
+#include "oneapi/dal/backend/interop/common.hpp"
+#include "oneapi/dal/backend/interop/table_conversion.hpp"
+
+namespace oneapi::dal::svm::backend {
+
+using std::int64_t;
+using dal::backend::context_cpu;
+
+namespace daal_svm             = daal::algorithms::svm;
+namespace daal_kernel_function = daal::algorithms::kernel_function;
+namespace interop              = dal::backend::interop;
+
+template <typename Float, daal::CpuType Cpu>
+using daal_svm_thunder_kernel_t = daal_svm::training::internal::
+    SVMTrainImpl<daal_svm::training::thunder, Float, daal_svm::Parameter, Cpu>;
+
+template <typename Float>
+static train_result call_daal_kernel(const context_cpu& ctx,
+                                     const descriptor_base& desc,
+                                     const table& data,
+                                     const table& labels,
+                                     const table& weights) {
+    const std::int64_t row_count    = data.get_row_count();
```

Comment:
nip: you have `using std::int64_t` on 25 line
```suggestion
    const int64_t row_count    = data.get_row_count();
```

Reply:
Done